railties/lib/rails/application/configuration.rb

```diff
@@ -605,6 +625,18 @@ def credentials_defaults
 
           { content_path: content_path, key_path: key_path }
         end
+
+        def generate_local_secret
+          key_file = Rails.root.join("tmp/local_secret.txt")
```

Comment:
Similar to `credentials_defaults` we can just call `root`:
```suggestion
          key_file = root.join("tmp/local_secret.txt")
```

Reply:
Nice, thanks!